### PR TITLE
fix(security): don't allow app enumeration via RevisionChartDetails

### PR DESCRIPTION
**Pull request by crenshaw-dev**
This endpoint was written before we had the fix in for app enumeration. I'm just applying the security fix to the additional endpoint.

### CODE REVIEW

File: server/application/application.go [hunk 1]
```diff
@@ -1371,17 +1371,12 @@ func (s *Server) RevisionMetadata(ctx context.Context, q *application.RevisionMe
...
 	if err != nil {
-		return nil, fmt.Errorf("error getting app by name: %w", err)
-	}
-	if err := s.enf.EnforceErr(ctx.Value("claims"), rbacpolicy.ResourceApplications, rbacpolicy.ActionGet, a.RBACName(s.ns)); err != nil {
-		return nil, fmt.Errorf("error enforcing claims: %w", err)
+		return nil, err
 	}
 	if a.Spec.Source.Chart == "" {
-		return nil, fmt.Errorf("no chart found for application: %v", appName)
+		return nil, fmt.Errorf("no chart found for application: %v", a.Name)
```
Comment (jannfis) [inline server/application/application.go:1379]: Error message should use a.QualifiedName() to cater for apps-in-any-namespace

**Review by jannfis — APPROVED** [review]
LGTM, with a tiny nit

File: server/application/application.go (hunk 1 above)
Comment (crenshaw-dev) [reply server/application/application.go]: Fixed!

**Pull request merged**